src/dictionaries/modifiers.js

```diff
@@ -40,6 +40,7 @@ const allModifiers = {
 	omit3: 'omit3',
 	omit5: 'omit5',
 	power: 'power',
+	alt: 'alt',
```

Comment:
here's the one that IMHO should show `7alt`

Reply:
It is actually, because the way the parser works is that the alt modifier will enforce a dominant 7th. So the rendering will always be 7(alt)